src/redux/books/books.js

```diff
@@ -0,0 +1,39 @@
+// Actions
+const ADD = 'bookstore-1/books/ADD';
+const UPDATE = 'bookstore-1/books/UPDATE';
+const DELETE = 'bookstore-1/books/DELETE';
+const LOAD = 'bookstore-1/books/LOAD';
+
+// Reducer
+export const reducer = (state = [], action = {}) => {
+  switch (action.type) {
+    case ADD: {
+      return state;
+    }
```

Comment:
- [ ] in case of action type `add`, the function should return a new state object in which the books array will contain a new book object, passed by `action.payload`. Kindly update the returned value of this case 👍